src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.c

```diff
@@ -44,188 +54,1324 @@ const ActorInit En_Dekubaba_InitVars = {
+void EnDekubaba_DrawStemExtended(EnDekubaba* this, PlayState* play) {
+    static Gfx* sStemDLists[] = { gDekuBabaStemTopDL, gDekuBabaStemMiddleDL, gDekuBabaStemBaseDL };
+    MtxF mf;
+    s32 i;
+    f32 scale;
+    f32 horizontalStepSize;
+    s32 stemSections;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B2890.s")
+    OPEN_DISPS(play->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B28B4.s")
+    if (this->actionFunc == EnDekubaba_PrunedSomersault) {
+        stemSections = 2;
+    } else {
+        stemSections = 3;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B2980.s")
+    scale = this->size * 0.01f;
+    Matrix_Translate(this->actor.world.pos.x, this->actor.world.pos.y, this->actor.world.pos.z, MTXMODE_NEW);
+    Matrix_Scale(scale, scale, scale, MTXMODE_APPLY);
+    Matrix_Get(&mf);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B29C4.s")
+    for (i = 0; i < stemSections; i++) {
+        mf.yw += 20.0f * Math_SinS(this->stemSectionAngle[i]) * this->size;
+        horizontalStepSize = Math_CosS(this->stemSectionAngle[i]) * 20.0f * this->size;
+        mf.xw -= horizontalStepSize * Math_SinS(this->actor.shape.rot.y);
+        mf.zw -= horizontalStepSize * Math_CosS(this->actor.shape.rot.y);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B2C40.s")
+        Matrix_Put(&mf);
+        Matrix_RotateZYX(this->stemSectionAngle[i], this->actor.shape.rot.y, 0, MTXMODE_APPLY);
+        gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(POLY_OPA_DISP++, sStemDLists[i]);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B2CB8.s")
+        Collider_UpdateSpheres(51 + 2 * i, &this->collider);
+        Collider_UpdateSpheres(52 + 2 * i, &this->collider);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B3044.s")
+        if (i == 0) {
+            if (this->actionFunc != EnDekubaba_Sway) {
+                this->actor.focus.pos.x = mf.xw;
+                this->actor.focus.pos.y = mf.yw;
+                this->actor.focus.pos.z = mf.zw;
+            } else {
+                this->actor.focus.pos.x = this->actor.home.pos.x;
+                this->actor.focus.pos.y = this->actor.home.pos.y + (40.0f * this->size);
+                this->actor.focus.pos.z = this->actor.home.pos.z;
+            }
+        }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B3078.s")
+        Matrix_MultZero(&this->bodyPartsPos[i + 1]);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B3170.s")
+    CLOSE_DISPS(play->state.gfxCtx);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dekubaba/func_808B3280.s")
+void EnDekubaba_DrawStemBasePruned(EnDekubaba* this, PlayState* play) {
```

Comment:
Looks like Prune is in use here. I think stick death drop should be called Prune :)